### PR TITLE
LPS-146533 [CP 2.0] Update Status on Activation Form Card - Analytics Cloud by users with Provisioning role

### PR DESCRIPTION
**Pull request by joanagrego**
### **How it was developed?**

Creating a dropdown in the Analytic's Cloud card status component for a staff user to set status from "In Progress" to "Active" after the onboarding is complete.
Only a staff user can access this possibility through the dropdown menu to change the status. To update the status, there are some conditions to create the Group ID field in the modal.

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ActivationStatus/AnalyticsCloud/index.js [hunk 1]
```diff
@@ -31,14 +41,26 @@ const ActivationStatusAnalyticsCloud = ({
...
 	const [activationStatusDate, setActivationStatusDate] = useState('');
 	const [isVisible, setIsVisible] = useState(false);
+	const [visible, setVisible] = useState(false);
 	const {observer, onClose} = useModal({
 		onClose: () => setIsVisible(false),
 	});
+	const {
+		observer: observerModalStatus,
+		onClose: onCloseModalStatus,
+	} = useModal({
```
Comment (iagochaves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ActivationStatus/AnalyticsCloud/index.js:54]: Declare a explicit name for for the first observer and onClose variables
```suggestion
	const {observer: ..., onClose: ...} = useModal({
		onClose: () => setIsVisible(false),
	});
	const {
		observer: observerModalStatus,
		onClose: onCloseModalStatus,
	} = useModal({
```

**Review by iagochaves — APPROVED** [review]

**Review by igordmf — APPROVED** [review]
Good job!

**Review by Lucas2691 — APPROVED** [review]
Nice job.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/styles/components/_activation-keys.scss [hunk 2]
```diff
@@ -33,3 +33,19 @@
...
+	border: 1px solid var(--color-neutral-2);
+	box-shadow: none;
+	left: 641px;
+	min-width: fit-content;
+	top: 276px;
+	ul li span {
+		padding: 0.75rem;
+		padding-right: 68px !important;
+	}
+}
```
[the remark this answers is not in the document]
Comment (joanagrego) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/styles/components/_activation-keys.scss:47]: Nice!

**Comment by joanagrego** [comment]
Resent -> https://github.com/pt-liferay-solutions/liferay-portal/pull/1073

**Pull request closed without merging**